app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -36,20 +36,38 @@ def items
     questions.map do |question|
       # We have answers as strings and hashes. The hashes need a little more
       # work to make them readable.
-      value = session[question].is_a?(Hash) ? concat_answer(session[question]) : session[question]
+      value = session[question].is_a?(Hash) ? concat_answer(session[question], question) : session[question]
 
       {
         field: t("coronavirus_form.questions.#{question}.title"),
-        value: value,
+        value: sanitize(value),
         edit: {
           href: question.dasherize,
         },
       }
     end
   end
 
-  def concat_answer(answer)
-    answer.values.compact.join(" ")
+  def concat_answer(answer, question)
+    concatenated_answer = []
+    joiner = " "
+
+    if question.eql?("contact_details")
```

Comment:
I realise that the following is quite write-everything-twice, but I think it's a touch clearer for future people to see what we're doing here.